currently getting this error when testing:

```
Warning: Command failed: /bin/sh -c cd app && npm install
npm WARN engine stacktrace-parser@0.1.1: wanted: {"node":"~0.10"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react-native does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer tcomb-form-native@0.1.7 wants react-native@^0.4.0

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Arian/git/crewapp/app/npm-debug.log
 Use --force to continue.
```

--- a/app/package.json
+++ b/app/package.json
@@ -6,7 +6,7 @@
     "start": "node_modules/react-native/packager/packager.sh"
   },
   "dependencies": {
-    "react-native": "^0.3.4",
+    "react-native": "^0.4.0",
     "react-sockets": "crewapp/react-native-sockets-io",
     "tcomb": "^1.0.1",
     "tcomb-form-native": "^0.1.6",

--- a/app/package.json
+++ b/app/package.json
@@ -6,7 +6,7 @@
     "start": "node_modules/react-native/packager/packager.sh"
   },
   "dependencies": {
-    "react-native": "^0.3.11",
+    "react-native": "^0.3.4",
     "react-sockets": "crewapp/react-native-sockets-io",
     "tcomb": "^1.0.1",
     "tcomb-form-native": "^0.1.6",

